tests/run/ext_type_none_arg.pyx

```diff
@@ -118,6 +118,18 @@ def ext_optional(x: typing.Optional[MyExtType], y: Optional[MyExtType]):
     """
     return attr(x) + attr(y)
 
+def ext_union(x: typing.Union[MyExtType, None], y: Union[None, MyExtType]):
```

Comment:
Again, a negative test would show that the type declarations are not just no-ops.